common/data_refinery_common/models/models.py

```diff
@@ -634,6 +744,24 @@ def calculate_size(self) -> None:
         self.size_in_bytes = os.path.getsize(self.absolute_file_path)
         return self.size_in_bytes
 
+    def delete_local_file(self):
+        """ Deletes a file from the path and actually removes it from the file system,
+        resetting the is_downloaded flag to false. Can be refetched from source if needed. """
```

Comment:
This comment looks like it was copied from the OriginalFile function because I don't think ComputedFile has  a `is_downloaded` field nor will they be re-fetched from the source.